src/Tables.jl

```diff
@@ -499,8 +499,18 @@ function Base.getproperty(sch::Schema{names, types}, field::Symbol) where {names
     if field === :names
         return names === nothing ? getfield(sch, :storednames) : names
     elseif field === :types
-        T = getfield(sch, :storedtypes)
-        return types === nothing ? (T !== nothing ? T : nothing) : Tuple(fieldtype(types, i) for i = 1:fieldcount(types))
+        if types === nothing
+            T = getfield(sch, :storedtypes)
+            return T !== nothing ? T : nothing
```

Comment:
I do not understand the lines in this branch. Why not just `return getfield(sch, :storedtypes)`?

Other than that I can merge this PR.

Reply:
🤷‍♂️ (I tried to make it as close as possible to the original code to get the PR merged faster)

But yeah let me change it!

Reply:
Done.